react/details/src/components/__tests__/Widget.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import 'components/_mocks/i18n';
+import Widget from 'components/Widget';
+import entityDataMockup from 'components/_mocks/entity-data.json';
```

Comment:
What about renaming `_mocks` to `__mocks__` (like the __tests__) one? I would do the same in my repo :)

Reply:
I was thinking about that, but went with midpoint as I have applied the save name for `_types`. What do you think about naming proptypes folder that way as well?

Also, about the mocks - we could add the ignore path option to the npm test script and move the mocks to the __tests__

Reply:
fine to have both __mocks__ and __types__.
think makes sense to have __mocks__ under __tests__ but I'm afraid we're complicating too much the folder structure